test/common/schemaform/actions.unit.spec.js

```diff
@@ -124,6 +124,36 @@ describe('Schemaform actions:', () => {
         });
       });
     });
+    it('should submit with auth header', () => {
+      const formConfig = {
+        chapters: {}
+      };
+      const form = {
+        pages: {
+          testing: {},
+        },
+        data: {
+          test: 1
+        }
+      };
+      global.sessionStorage = { userToken: 'testing' };
```

Comment:
I'm assuming it has something to do with running as a unit test, but why `global` here?